Garage 2.0/Controllers/VehiclesController/VehiclesController.cs

```diff
@@ -12,14 +12,15 @@ namespace Garage_2._0.Controllers.VehiclesController
     public class VehiclesController : Controller
     {
         private readonly GarageVehicleContext _context;
-
         Vehicle[] parkingSpots;
-
-        IConfiguration _iConfig;
-        public VehiclesController(GarageVehicleContext context, IConfiguration iConfig)
+        IConfiguration _config;
+        public VehiclesController(GarageVehicleContext context,IConfiguration config)
         {
             _context = context;
-            _iConfig = iConfig;
+            _config = config;
+            Global.Garagecapacity = _config.GetValue<int>("GarageCapacity:Capacity");
+            Global.HourlyRate = _config.GetValue<double>("Price:HourlyRate");
```

Comment:
You can remove the _config = config on line 20 and replace _config with config on lines 21 and 22.

Reply:
removed